### PR TITLE
Fixed #159 -- Used TracSpamFilter compatible with 1.4.

### LINKED ISSUE #159: TracSpamFilter crashes.
https://code.djangoproject.com/admin/spamfilter/bayes:
```
File "/venv/lib/python2.7/site-packages/trac/web/main.py", line 639, in dispatch_request          
  dispatcher.dispatch(req)
File "/venv/lib/python2.7/site-packages/trac/web/main.py", line 271, in dispatch          
  method=method)
File "/venv/lib/python2.7/site-packages/trac/web/chrome.py", line 1447, in render_template          
  fragment, iterable, method)
File "/venv/lib/python2.7/site-packages/trac/web/chrome.py", line 1501, in _render_jinja_template          
  domain)
File "/venv/lib/python2.7/site-packages/trac/web/chrome.py", line 1611, in prepare_template          
  template = self._load_jinja_template(filename, text)
File "/venv/lib/python2.7/site-packages/trac/web/chrome.py", line 1402, in _load_jinja_template          
  return (self.jenv_text if text else self.jenv).get_template(filename)
File "/venv/lib/python2.7/site-packages/jinja2/environment.py", line 883, in get_template          
  return self._load_template(name, self.make_globals(globals))
File "/venv/lib/python2.7/site-packages/jinja2/environment.py", line 857, in _load_template          
  template = self.loader.load(self, name, globals)
File "/venv/lib/python2.7/site-packages/jinja2/loaders.py", line 127, in load          
  code = environment.compile(source, name, filename)
File "/venv/lib/python2.7/site-packages/jinja2/environment.py", line 638, in compile          
  self.handle_exception(source=source_hint)
File "/venv/lib/python2.7/site-packages/jinja2/environment.py", line 832, in handle_exception          
  reraise(*rewrite_traceback_stack(source=source))
File "/venv/lib/python2.7/site-packages/tracspamfilter/templates/admin_bayes.html", line 31, in template          
  disabled="${dblinesreducenum &lt; 10000 or None}" />
```

We should try to use 1.4.x version.

### PR DESCRIPTION
**Pull request by felixxm**
https://trac.edgewall.org/wiki/SpamFilter#GetthePlugin

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ libsass==0.17.0
 Genshi==0.7.7  # still required by some plugins
 
 # Trac plugins
-TracSpamFilter==1.2.7
+svn+https://svn.edgewall.org/repos/trac/plugins/1.4/spam-filter
```
Comment (bmispelon) [inline requirements.txt:12]: You can install the zip directly, that way you don't need subversion as a requirement in the Dockerfile
```suggestion
https://trac.edgewall.org/browser/plugins/1.4/spam-filter?format=zip
```

**Review by bmispelon — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -12,6 +12,7 @@ RUN set -x \
     make \
     postgresql-client \
     python2.7 \
+    subversion \
```
Comment (felixxm) [inline Dockerfile:15]:
```suggestion
```

**Comment by felixxm** [comment]
Updated :+1:

**Review by bmispelon — APPROVED** [review]

**Comment by bmispelon** [comment]
The new requirements file installs without error for me locally, so I think we're good to ship 🏎️

**Pull request merged**